pkg/admission/provider.go

```diff
@@ -1,93 +1,91 @@
 // TODO: only supports plain text with no auth, add security later
-// TODO: use keystone to access Neutron API
+// TODO: check keystone expiration date 
 package admission
 
 import (
-	resty "gopkg.in/resty.v1"
+	"fmt"
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack"
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/networks"
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/ports"
+)
+
+const (
+	providerDefaultRegion  = "RegionOne"
+	providerNeutronService = "neutron"
+	providerVersion        = "v2.0"
```

Comment:
I would remove the const providerVersion and let the user specify this as part of the provider-url / OS_AUTH_URL, in combination with the additional  OS_PROJECT_NAME and OS_DOMAIN_NAME this would enable keystone v3.

Reply:
Seems like good idea. Thanks